worker/docker.go

```diff
@@ -21,6 +21,7 @@ const containerModelDir = "/models"
 const containerPort = "8000/tcp"
 const pollingInterval = 500 * time.Millisecond
 const containerTimeout = 2 * time.Minute
+const optFlagsTimeout = 5 * time.Minute // Pipelines with optimization flags can take longer to start.
```

Comment:
I just noticed that this timeout doesn't fully work. Will fix this tonight.

Reply:
You're right, I overlooked that one! Thanks. I'll fix it after the watercooler chat. 👍🏻